website/docs/r/compute_fleet.html.markdown

```diff
@@ -0,0 +1,740 @@
+
+* `configuration_blob_uri` - (Optional) Specifies the URI to an azure blob that will replace the default configuration for the package if provided. Changing this forces a new resource to be created.
+
+* `order` - (Optional) Specifies the order in which the packages have to be installed. Defaults to `0`. Changing this forces a new resource to be created.
+
+* `tag` - (Optional) Specifies a passthrough value for more generic context. Changing this forces a new resource to be created.
+
+* `treat_failure_as_deployment_failure_enabled` - (Optional) Whether to treat any failure for any operation in the virtual machine application as a deployment failure. Defaults to `false`. Changing this forces a new resource to be created.
+
+---
+
+A `identity` block supports the following:
+
+* `type` - (Required)  The type of managed identity that should be configured on the Compute Fleet. The only possible value is `UserAssigned`.
+
+* `identity_ids` - (Optional) Specifies a list of user managed identity IDs to be assigned to the Compute Fleet.
+
+---
+
+A `ip_configuration` block supports the following:
+
+* `name` - (Required) The Name which should be used for the IP configuration. Changing this forces a new resource to be created.
+
+* `subnet_id` - (Required) The ID of the Subnet which the IP configuration should be connected to. Changing this forces a new resource to be created.
+
+* `application_gateway_backend_address_pool_ids` - (Optional) Specifies a list of backend address pools IDs from an application gateway which the Compute Fleet should be connected to. Changing this forces a new resource to be created.
+
+* `application_security_group_ids` - (Optional) Specifies a list of application security group IDs which the Compute Fleet should be connected to. Changing this forces a new resource to be created.
+
+* `load_balancer_backend_address_pool_ids` - (Optional) Specifies a list of backend address pools IDs from a load balancer which the Compute Fleet should be connected to. Changing this forces a new resource to be created.
+
+* `primary_ip_configuration_enabled` - (Optional) Whether to set it as the primary IP configuration. Defaults to `false`. Changing this forces a new resource to be created.
+
+* `public_ip_address` - (Optional) One or more `public_ip_address` blocks as defined below. Changing this forces a new resource to be created.
+
+* `version` - (Optional) The internet protocol version which should be used for the IP configuration. Possible values are `IPv4` and `IPv6`. Defaults to `IPv4`. Changing this forces a new resource to be created.
+
+---
+
+A `ip_tag` block supports the following:
+
+* `tag` - (Required) The IP tag associated with the public IP. Changing this forces a new resource to be created.
+
+* `type` - (Required) The type of IP tag. Changing this forces a new resource to be created.
```

Comment:
type? what are the possible values of type?

i would have expected it to be tag.key and tag.value?

Reply:
I think it makes sense to understand that "ipTagType is the key of the tag, and tag is the value", but from the perspective of Azure's data structure semantics and API design intent, they are not key-value pairs in the traditional sense, but a combination of `purpose category` + `purpose name`.

For example:
```
"ipTags": [
{
"ipTagType": "RoutingPreference",
"tag": "Internet"
}
]

"ipTags": [
{
"ipTagType": "RoutingPreference",
"tag": "Microsoft"
}
]
```
The above IP addresses uses routing preference (RoutingPreference), specifically Internet/Microsoft routing.

Therefore, I assume that these two sets of fields together describe the multiple dimensional uses of this IP, rather than a pure tags mapping. What about you?

Reply:
Well is any key/value possible or is there only a subset that are valid?

Reply:
Hi @SahilArora92 for the question above, "Well is any key/value possible or is there only a subset that are valid?", could you please help me check it?

If it is not any key/value possible, could you please help me confirm all possible values ​​​​for [ipTagType](https://github.com/Azure/azure-rest-api-specs/blob/0bf123645e53f9006c15a962087c9073d7fd455f/specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json#L3009) and [tag](https://github.com/Azure/azure-rest-api-specs/blob/0bf123645e53f9006c15a962087c9073d7fd455f/specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json#L3013) so that I could write it into the document? Thanks in advance!

Reply:
Hi @sinbai There is no specific key/value combinations here. These is a dynamic key-value pair.

IPTagType can have values like NetworkDomain, FirstPartyUsage, RoutingPreference etc.

Reply:
> Hi @sinbai There is no specific key/value combinations here. These is a dynamic key-value pair.
> 
> IPTagType can have values like NetworkDomain, FirstPartyUsage, RoutingPreference etc.

Hi @ykh015 thanks for your response, can I confirm that you are a member of the service team, right? Can I consider this response official?

Reply:
Yes, I am the member of the service team.

Reply:
Hi @katbyte could you please take a look at the response from the service team? If there are any changes needed, please feel free to let me know. Thank you!